fe1-web/src/features/wallet/objects/Wallet.ts

```diff
@@ -43,3 +44,44 @@ export async function recoverWalletPoPTokens(): Promise<Record<string, Record<st
 
   return tokens;
 }
+/**
+ * Recovers all PoP tokens associated with this wallet.
+ * @return Promise<RollCallToken[]>
+ * @remarks
+ * This is implemented by checking through all known Roll Calls of all known LAOs,
+ * generating tokens for them and checking if the token is a verified attendee.
+ */
+export async function recoverWalletRollCallTokens(): Promise<RollCallToken[]> {
```

Comment:
You could also pass the roll calls as input to this function, would make testing a bit easier :)